src/app/schemas.py

```diff
@@ -0,0 +1,22 @@
+"""Author: Tadd Bindas"""
+
+from pydantic import BaseModel
+
+
+class TRouteOuput(BaseModel):
```

Comment:
TRouteOutput is perhaps a misleading name implying this class/return contains actually routed output data.  I would rename this to something like `TRouteStatus` since this is really just status messaging about the run, but doesn't contain model output.